### ISSUE TITLE
CRC fails to start -- Windows 10 -- Error parsing time when validating certificates

### ISSUE THREAD

**Issue by pthopa** [code-ready/crc#1589]
### General information

  * OS: Windows 10 Enterprise Version 1803
  * Hypervisor: Hyper-V 
  * Did you run `crc setup` before starting it (Yes/No)? Yes

## CRC version
```bash
# Put the output of `crc version`
```
  CodeReady Containers version: 1.17.0+99f5c87
OpenShift version: 4.5.14 (embedded in binary)

## CRC status
```bash
# Put the output of `crc status`
```
ERRO Cannot get root partition usage: strconv.ParseInt: parsing "": invalid syntax
Cannot get root partition usage: strconv.ParseInt: parsing "": invalid syntax

## CRC config
```bash
# Put the output of `crc config view`
```
No output

## Host Operating System
```bash
# Put the output of `cat /etc/os-release` in case of Linux
# put the output of `sw_vers` in case of Mac
# Put the output of `systeminfo` in case of Windows
```
OS Name:                   Microsoft Windows 10 Enterprise
OS Version:                10.0.17134 N/A Build 17134
OS Manufacturer:           Microsoft Corporation
OS Configuration:          Member Workstation
OS Build Type:             Multiprocessor Free
System Manufacturer:       Dell Inc.
System Model:              Latitude 5500
System Type:               x64-based PC
Processor(s):              1 Processor(s) Installed.
                               [01]: Intel64 Family 6 Model 142 Stepping 12 GenuineIntel ~1910 Mhz
BIOS Version:              Dell Inc. 1.6.5, 12/26/2019
Windows Directory:         C:\WINDOWS
System Directory:          C:\WINDOWS\system32
Boot Device:               \Device\HarddiskVolume2
System Locale:             en-us;English (United States)
Input Locale:              en-us;English (United States)
Time Zone:                 (UTC-05:00) Eastern Time (US & Canada)
Total Physical Memory:     32,543 MB
Available Physical Memory: 14,738 MB
Virtual Memory: Max Size:  37,407 MB
Virtual Memory: Available: 18,994 MB
Virtual Memory: In Use:    18,413 MB

### Steps to reproduce

  1.  crc cleanup
  2.  crc setup
  3. crc start
  4. 
up
### Expected
crc would start and be in running state

### Actual
crc failed to start
INFO Extracting bundle: crc_hyperv_4.5.14.crcbundle ...
crc.vhdx: 12.90 GiB / 12.90 GiB [-----------------------------------------------------------------------------] 100.00%
INFO Checking size of the disk image C:\Users\thopap\.crc\cache\crc_hyperv_4.5.14\crc.vhdx ...
INFO Creating CodeReady Containers VM for OpenShift 4.5.14...
INFO CodeReady Containers VM is running
INFO Generating new SSH Key pair ...
INFO Copying kubeconfig file to instance dir ...
INFO Starting network time synchronization in CodeReady Containers VM
INFO Verifying validity of the cluster certificates ...
**ERRO Failed to check certificate validity: parsing time "" as "2006-01-02T15:04:05Z07:00": cannot parse "" as "2006"
Failed to check certificate validity: parsing time "" as "2006-01-02T15:04:05Z07:00": cannot parse "" as "2006"**

### Logs

You can start crc with `crc start --log-level debug` to collect logs.
Please consider posting this on http://gist.github.com/ and post the link in the issue.
https://gist.github.com/pthopa/cae422d3e53bfe08c56de65edc5ba8cc

**Comment by gbraad** [comment]
Duplicate of #1549. Fixed

**Comment by gbraad** [comment]
This will certainly be in the next release that is available in about 2 weeks. You can however see the previously mentioned issue and get a build from master (but this is unsupported for general use as a few additional steps are needed, like bundle location, etc).
